test/components/project/enable-encryption.spec.js

```diff
@@ -129,9 +129,42 @@ describe('ProjectEnableEncryption', () => {
   });
 
   describe('after a successful response', () => {
-    const submit = () => {
```

Comment:
It's a little hard to see with the diff, but there used to be a helper function named `submit()` that called `load()`. However, the new test couldn't use `submit()`, because it needs to set the input twice. And actually, only one test needed to be using `load()`. I changed the other existing test (about the icon) so that it calls `mockHttp()` instead.